assets/js/search.js

```diff
@@ -0,0 +1,124 @@
+(function() {
+    'use stict';
+    var activityCards, find, mapCardToClassName, mapCardToHeading, hideCard,
```

Comment:
Is there a reason for the variables at the top or is this just Crockford's advise? ;)